src/Raven.Server/Documents/Indexes/Index.cs

```diff
@@ -4135,7 +4162,7 @@ public enum CanContinueBatchResult
             RenewTransaction
         }
 
-        public CanContinueBatchResult CanContinueBatch(IndexingStatsScope stats, QueryOperationContext queryContext, TransactionOperationContext indexingContext,
+        public CanContinueBatchResult CanContinueBatch(IndexingStatsScope stats, IndexingWorkType workType, QueryOperationContext queryContext, TransactionOperationContext indexingContext,
```

Comment:
At this point, better to create a struct and pass it using `in`

Reply:
I don't follow. Use struct instead of enum? What's the reasoning?

Reply:
You mean to create a struct like `CanContinueBatchParameters` and put all options there?

Reply:
Introduced in https://github.com/ravendb/ravendb/pull/12710/commits/3463878bcdcbf4e66654f135e28da980bfa579b3